src/lib/stores/item.svelte.ts

```diff
@@ -0,0 +1,83 @@
+import { v7 as uuid } from 'uuid'
+import dayjs from 'dayjs'
+import { possibleItems } from '$lib/components/item/itemGenerator'
+import { randomIntFromInterval } from '$lib/utils'
+import type { StoredItem } from '$lib/types'
+
+export function createItemStore() {
```

Comment:
this is how a store looks.
we use getters to not lose reactivity (the docs say destructuring loses reactivity)